Packages/MIES/MIES_Configuration.ipf

```diff
@@ -0,0 +1,1785 @@
+#pragma TextEncoding = "UTF-8"
+#pragma rtGlobals=3 // Use modern global access method and strict wave access.
+#pragma rtFunctionErrors=1
+
+#ifdef AUTOMATED_TESTING
+#pragma ModuleName=MIES_CONF
+#endif
+
+/// @file MIES_Configuration.ipf
+///
+/// @brief __CONF__ Import user settings to configure paramters for Ephys experiments
+
+#if exists("MCC_GetMode") && exists("AxonTelegraphGetDataStruct")
+#define AMPLIFIER_XOPS_PRESENT
+#endif
+
+//******************************************************************************************************************************
+//
+// The Configuration module allows to save and load the GUI state of complete windows including its subwindows
+//
+// The main functions are CONF_SaveWindow and CONF_RestoreWindow. Special functions are included for the
+// DA_Ephys panel that include additional initialization: CONF_SaveDAEphys and CONF_RestoreDAEphys.
+//
+// The GUI state is saved in a text file in json format. In the basic structure of the file the main blocks
+// are the windows and subwindows and within are the control blocks.
+// A bitmask parameter allows to configure which control information is saved. See WindowControlSavingMask
+// Currently supported is Value, Position/Size, Userdata, Disabled state and Type of a control.
+// Not implemented is e.g. Color/Background Color
+// As Value only the V_Value and/or S_Value and/or S_DataFolder is saved. Special control properties for e.g.
+// ListBox like V_SelCol are currently not implemented.
+//
+// How GUI controls are handled can be controlled through userdata settings of each control. The following
+// userdata settings are supported:
+// - Config_NiceName: sets a human readable name of the control used in the configuration file
+//                    This name may not be a duplicate of another controls name or ControlArray name
+// - Config_JSONPath: Sets a block name in which the control appears in the configuration file.
+//                    If not set then "Generic" is used. The name can not contain the '/' character.
+// - Config_DontSave: When set to 1 the control is ignored on saving the window.
+// - Config_DontRestore: When set to 1 the control is ignored on restoring the window.
+// - Config_PushButtonOnRestore: Only valid with Button controls. When the button is restored a button press
+//                               is issued.
+// - Config_RestorePriority: Sets a number that defines the order in which controls get restored. Lower numbers
```

Comment:
@t-b can decimals be used for `Config_RestorePriority`? I'm thinking of the case where I forget about a control and later want to weave it in between other controls without needing to change existing priority settings.

Reply:
Config_RestorePriority is double precision, I will add that to the brief docu.
I also organized the current priorities roughly in steps of 10.